catalog.html

```diff
@@ -1,171 +1,190 @@
+              <label for="check-3" class="filter-name">
                 <input type="checkbox" name="check-3" id="check-3">
                 <span>Wi-Fi</span>
               </label>
             </li>
           </ul>
         </fieldset>
-         <fieldset>
-          <legend>Тип жилья:</legend>
-          <ul>
-            <li>
-              <label for="check-4">
+        <fieldset class="filter-block">
+          <legend class="filter-title">Тип жилья:</legend>
+          <ul class="filter-list">
+            <li class="filter-item">
+              <label for="check-4" class="filter-name">
                 <input type="checkbox" checked name="check-4" id="check-4">
                 <span>Гостиница</span>
               </label>
             </li>
-            <li>
-              <label for="check-5">
+            <li class="filter-item">
+              <label for="check-5" class="filter-name">
                 <input type="checkbox" checked name="check-5" id="check-5">
                 <span>Мотель</span>
               </label>
             </li>
-             <li>
-              <label for="check-6">
+            <li class="filter-item">
+              <label for="check-6" class="filter-name">
                 <input type="checkbox" checked name="check-6" id="check-6">
                 <span>Апартаменты</span>
               </label>
             </li>
           </ul>
         </fieldset>
-        <fieldset>
-          <legend>Стоимость в сутки (Р):</legend>
+        <fieldset class="filter-range">
+          <legend class="filter-title">Стоимость в сутки (Р):</legend>
           <input type="range">
-          <button type="submit">Показать</button>
+          <button type="submit" class="filter-button">Показать</button>
         </fieldset>
       </form>
-      </section>
-      <section class="content">
-        <div class="sort-top">
-          <h2>Найдено: 3</h2>
-          <ul>
-            <li>Сортировка:</li>
-            <li class="active"><a>По цене</a></li>
-            <li><a href="#!">По типу</a></li>
-            <li><a href="#!">По рейтингу</a></li>
+    </section>
+    <section class="page-content">
+      <div class="sort-top">
+        <div class="container">
+          <h2 class="sort-title">Найдено: 3</h2>
+          <ul class="sort-list">
+            <li class="sort-list-item">Сортировка:</li>
+            <li class="sort-list-item active"><a>По цене</a></li>
+            <li class="sort-list-item"><a href="#!">По типу</a></li>
+            <li class="sort-list-item"><a href="#!">По рейтингу</a></li>
           </ul>
-          <ul>
+          <ul class="sort-list-button">
             <li class="active"><a>По убыванию</a></li>
             <li><a href="#!">По возростанию</a></li>
           </ul>
         </div>
-        <div class="sort-rezult">
-          <article>
-            <div>
+      </div>
+      <div class="sort-rezult">
+        <article class="sort-rezult-item">
```

Comment:
article - самостоятельная и независимая часть документа, страницы, приложения или сайта. Поэтому класс для article и его элементов должен быть соответствующим (название никак не завязано на внешнее окружение). Хорошим названием будет:
 ```suggestion
         <article class="hotel-card">
```